src/command_modules/azure-cli-dms/azure/cli/command_modules/dms/_help.py

```diff
@@ -44,7 +44,7 @@
     examples:
         - name: Create an instance of DMS.
           text: >
-            az dms create -l westus -n mydms -g myresourcegroup --sku-name Basic_2vCores --subnet /subscriptions/{vnet subscription id}/resourceGroups/{vnet resource group}/providers/Microsoft.Network/virtualNetworks/{vnet name}/subnets/{subnet name} --tags tagName1=tagValue1 tagWithNoValue
+            az dms create -l westus -n mydms -g myresourcegroup --sku-name Basic_2vCores --subnet "/subscriptions/{vnet subscription id}/resourceGroups/{vnet resource group}/providers/Microsoft.Network/virtualNetworks/{vnet name}/subnets/{subnet name}" --tags tagName1=tagValue1 tagWithNoValue
```

Comment:
Here I would replace the "things with spaces" with their "camelCaseEquivalent".  What error will users see if this is not quoted?